src/index.js

```diff
@@ -17,6 +17,9 @@ function generateNewStylesheet(stylesheet) {
       else if (key === 'background') {
         newStyle.backgroundColor = value;
       }
+      else if (key === 'display') {
+        return newStyle
```

Comment:
can you add semicolon, i'll try to get an eslint or convert to prettier soon to avoid these annoyances